Source/Renderer/Texture.js

```diff
@@ -281,13 +283,13 @@ define([
                         magFilter = TextureMagnificationFilter.NEAREST;
                     }
 
-                    sampler = {
+                    sampler = new Sampler({
```

Comment:
In general, we don't provide parameters when they are the same as the default value.